libs/portals/admin/application-system/src/module.tsx

```diff
@@ -23,14 +28,31 @@ const getScreen = ({ userInfo }: PortalModuleRoutesProps): React.ReactNode => {
 export const applicationSystemAdminModule: PortalModule = {
   name: m.applicationSystem,
   layout: 'full',
-  enabled: ({ userInfo }) => {
-    return userInfo.scopes.some((scope) => allowedScopes.includes(scope))
-  },
+  enabled: ({ userInfo }) =>
+    userInfo.scopes.some((scope) => allowedScopes.includes(scope)),
```

Comment:
The statistics should only be available to users with the `applicationSystemAdmin` scope

We have to find a way to exclude Statistics from the view if the user does not have `applicationSystemAdmin`

Reply:
This should work

```ts
routes: (props) => [
  // .....
  {
    name: m.statistics,
    path: ApplicationSystemPaths.Statistics,
    element: <Statistics />,
    enabled: props.userInfo.scope?.includes(AdminPortalScope.applicationSystemAdmin),
  },
]
```